ext/native/thin3d/thin3d_vulkan.cpp

```diff
@@ -1251,10 +1251,14 @@ class VKFramebuffer : public Framebuffer {
 	// Inherits ownership so no AddRef.
 	VKFramebuffer(VKRFramebuffer *fb) : buf_(fb) {}
 	~VKFramebuffer() {
-		buf_->Release();
+		buf_->vulkan_->Delete().QueueCallback([](void *fb) {
+			VKRFramebuffer *vfb = static_cast<VKRFramebuffer *>(fb);
+			delete vfb;
+		}, buf_);
 	}
 	VKRFramebuffer *GetFB() const { return buf_; }
 private:
+	VulkanContext *vulkan_;  // Unfortunate to have to keep this in each VKFramebuffer.
```

Comment:
Where is this set?  Looks like you're going to buf_ instead.

-[Unknown]

Reply:
Oh, I passed it in before, just forgot to remove it when I realized I could just steal the pointer from buf_. Though it's not pretty :P